api/src/main/java/org/pmiops/workbench/actionaudit/auditors/AdminAuditor.java

```diff
@@ -13,4 +13,8 @@ void fireViewNotebookAction(
   void fireLockWorkspaceAction(long workspaceId, AdminLockingRequest adminLockingRequest);
 
   void fireUnlockWorkspaceAction(long workspaceId);
+
+  void firePublishWorkspaceAction(long workspaceId, String category);
+
+  void fireUnPublishWorkspaceAction(long workspaceId, String category);
```

Comment:
lowercase p

Reply:
Sorry about this should have updated in the last commit